controllers/player.controller.js

```diff
@@ -1,43 +1,69 @@
 const Player = require('../models/player.model')
+const logs = require('../helpers/logsHelper')
 
-exports.hello = (req, res) => {
-  res.send('Hello world')
+exports.playerCreate = async (req, res) => {
```

Comment:
this is updating not creating, right ?

Reply:
no. This is to check if the player exists, and if not will create one.